test/definitions/ohi/linux/cassandra-suse.json

```diff
@@ -0,0 +1,52 @@
+{
+  "global_tags": {
+    "owning_team": "OpenSource",
+    "Environment": "development",
+    "Department": "Product",
+    "Product": "Virtuoso"
+  },
+
+  "resources": [{
+    "id": "host1",
+    "display_name": "SuseInfraCassandraInstallHost",
+    "provider": "aws",
+    "type": "ec2",
+    "size": "t2.small",
+    "ami_name": "openSUSE 15.2 Leap*"
+  }],
+
+  "services": [{
+    "id": "cassandra1",
+    "destinations": ["host1"],
+    "source_repository": "https://github.com/newrelic/open-install-library.git",
+    "deploy_script_path": "test/deploy/linux/cassandra/install/suse/roles",
+    "port": 80
```

Comment:
I don't believe we need port 80 to be opened for Cassandra. You should use the port 9999 instead (port is a required field, the value of 9999 is equivalent to no-port-required for this service).